tekton/pipeline/data-pipeline-v6.yaml

```
@@ -0,0 +1,218 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: rag-data-pipeline-v6
+spec: 
+      params:
+      - name: repo
+        type: string
+        default: git@github.com:ilan-pinto/lightspeed-rag-content.git
+        description: Plain text source repo git 
+      - name: branch-name
+        type: string
+        description: The git branch to clone.
+        default: main
+
+      - name: input-folders
+        description: List of folders for processing.
+        type: array
+        default: ["/workspace/source/ocp-product-docs-plaintext/4.14","/workspace/source/ocp-product-docs-plaintext/4.15"]
+
+      - name: include-evalution
+        type: string
+        default: "true"
+        description: run auto evaluation 
+
+      - name: relese
+        type: string
+        default: "true"
+        description: release artifact true/false
+
+      - name: question-folder
+        type: string
+        description: Plain text question-folder path
+        default: /workdir/questions/openai-gpt-3.5-turbo_no_eval.json
+
+      - name: tag
+        type: string
+        description: rag results base tag
+        default: "chunk-10-overlap-10"
```

```suggestion
        default: "chunk-512-overlap-50"
```

?  based on the defaults below, anyway.....